order.html

```diff
@@ -0,0 +1,179 @@
+              <a href="./order.html" class="nav-link">Login</a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+    <main>
+      <section class="form-section">
+        <div class="form-container">
+          <h1>Order your Karma wifi device today!</h1>
+          <form action="">
+            <div class="oneline-input">
+              <div class="borders">
+                <label for="fname"
+                  >First name<span class="required">*</span></label
+                >
+                <input
+                  id="fname"
+                  type="text"
+                  name="fname"
+                  minlength="3"
+                  maxlength="10"
+                  required
+                />
+              </div>
+              <div class="borders">
+                <label for="lname"
+                  >Last name<span class="required">*</span></label
+                >
+                <input
+                  id="lname"
+                  type="text"
+                  name="lname"
+                  minlength="3"
+                  maxlength="10"
+                  required
+                />
+              </div>
+            </div>
+
+            <div class="borders">
+              <label for="address"
+                >Address<span class="required">*</span></label
+              >
+              <input
+                id="address"
+                class="address"
+                type="text"
+                name="address"
+                required
+              />
+            </div>
+            <div class="borders">
+              <label for="address2">Address 2</label>
+              <input
+                id="address2"
+                class="address"
+                type="text"
+                name="address2"
+              />
+            </div>
+            <div class="oneline-input">
+              <div class="borders">
+                <label for="city">City<span class="required">*</span></label>
+                <select id="city" name="city">
+                  <option selected="selected" value="">
+                    Select your city...
+                  </option>
+                  <option value="florida">London</option>
+                  <option value="florida">Birmingham</option>
+                  <option value="florida">Paris</option>
+                  <option value="florida">New York</option>
+                  <option value="florida">Florida</option>
+                  <option value="texas">Texas</option>
+                </select>
+              </div>
+              <div class="borders">
+                <label for="postcode"
+                  >Postcode<span class="required">*</span></label
+                >
+                <input
+                  id="postcode"
+                  class="postcode"
+                  type="text"
+                  name="postcode"
+                  required
+                />
+              </div>
+            </div>
```

Comment:
All the above inputs should be wrapped inside a `<fieldset>` with a `<legend`> to signify that they belong to the same 'topic' i.e. they all refer to the user's contact details. 
The following radio buttons on the other hand deal with details relating to the product that is being bought, so could also have its own `<fieldset>` and `<legend`>